### PR TITLE
refactor(modules): ES5 require statements converted to ES2015 import statments

### PR DESCRIPTION
**Pull request by farhan787**
issue #532

### CODE REVIEW

**Comment by farhan787** [comment]
@shane-tomlinson here is the step-2.

**Comment by farhan787** [comment]
> 1. There are still a log of `require` statements that can be converted

@shane-tomlinson I only did the changes under app/ directory, I'm changing these too.

File: packages/fxa-content-server/app/scripts/views/settings.js [hunk 1]
```diff
@@ -4,38 +4,38 @@
...
+import ClientDisconnectView from './settings/client_disconnect';
+import ClientsView from './settings/clients';
+import Cocktail from 'cocktail';
+import CommunicationPreferencesView from './settings/communication_preferences';
+import DeleteAccountView from './settings/delete_account';
+import DisplayNameView from './settings/display_name';
+import Duration from 'duration';
+import EmailsView from './settings/emails';
+import LoadingMixin from './mixins/loading-mixin';
 require('modal');
```
[the remark this answers is not in the document]
Comment (farhan787) [reply packages/fxa-content-server/app/scripts/views/settings.js]: yes, it should be.

**Comment by farhan787** [comment]
> ./scripts/lib/constants.js:const popularEmailDomains = require('./popular-email-domains.json');
> ./scripts/lib/jwcrypto-shim.js:require('jwcrypto.rs');
> ./scripts/lib/mailcheck.js:require('mailcheck');
> ./scripts/lib/experiments/base.js:const _ = require('underscore');
> ./scripts/lib/experiments/base.js:const Backbone = require('backbone');
> ./scripts/lib/experiments/base.js:const NotifierMixin = require('lib/channels/notifier-mixin');
> ./scripts/lib/experiments/base.js:const Storage = require('lib/storage');
> ./scripts/lib/jquery.js:window.jQuery = window.$ = require('jquery');

@shane-tomlinson if I'm changing require to import under lib/ directory then it's saying that unknown identifier because of backend, I think these can't be converted.

File: packages/fxa-content-server/app/scripts/views/sms_send.js [hunk 2]
```diff
@@ -9,21 +9,21 @@
...
-const PairingGraphicsMixin = require('./mixins/pairing-graphics-mixin');
+import AuthErrors from '../lib/auth-errors';
+import Cocktail from 'cocktail';
+import CountryTelephoneInfo from '../lib/country-telephone-info';
+import { FIREFOX_MOBILE_INSTALL } from '../lib/sms-message-ids';
+import FlowEventsMixin from './mixins/flow-events-mixin';
+import FormPrefillMixin from './mixins/form-prefill-mixin';
+import FormView from './form';
+import PairingGraphicsMixin from './mixins/pairing-graphics-mixin';
 const { MARKETING_ID_AUTUMN_2016, SYNC_SERVICE } = require('../lib/constants');
```
[the remark this answers is not in the document]
Comment (farhan787) [reply packages/fxa-content-server/app/scripts/views/sms_send.js]: Yes

**Comment by farhan787** [comment]
> ./tests/test_start.js:require('./spec/head/startup-styles');

@shane-tomlinson for ./tests/test_start.js:require statements [see this](https://github.com/mozilla/fxa/blob/e09cd808d36f586dd0021ac58a1c371b048068a5/packages/fxa-content-server/app/tests/test_start.js#L11)

**Comment by shane-tomlinson** [comment]
Replaced by #756, closing.

**Pull request closed without merging**